FrontEnd/src/components/basics/ImageDropdown/ImageDropdown.tsx

```diff
@@ -3,18 +3,14 @@ import BootstrapDropdown, * as BootstrapDropdownMetadata from 'react-bootstrap/D
 import styles from './ImageDropdown.module.scss';
 import { Button } from '@basics';
 import cn from 'classnames';
-import { useRouter } from 'next/router';
-
-const ButtonWrapper: FunctionComponent = ({ children }) => (
-  <Button variant="wrapper">{children}</Button>
-);
 
 // TODO expand this
 interface itemConfig {
   href?: string;
   label: string;
   labelClassName?: string;
   onClick?: () => void;
+  selected: boolean;
```

Comment:
Should the "selected" prop be optional or required?

Reply:
required. it needs to be explicitly put in

Reply:
Would it not be better to make it optional so people only have to put selected when it should be selected? Think beyond the navbar, there might be cases where we might not need the `selected` prop at all. Making it optional would make it so that people would never have to do `selected={false}`. What are your thoughts?